plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -56,9 +56,10 @@ def update_replica_count(new_count)
 
       if replicas_live >= replica_target
         self.status ='live'
-      elsif replicas_live > 0
+      elsif replicas_live > 0 && !failed?
```

Comment:
I'm thinking if this should be a safe guard at the beginning of the method, I mean, should we return right away if the "release_doc" has failed?

```
def update_replica_count(new_count)
  return if failed?
  ...
end
```

Because as soon as a pod fails, this status should never be updated again, I think.